File: src/main/java/com/skidders/sigma/module/Category.java
package com.skidders.sigma.module;

public enum Category {
    GUI("Gui"),
    MOVEMENT("Movement"),
    RENDER("Render"),
    COMBAT("Combat"),
    MISC("Misc"),
    PLAYER("Player"),
    ITEM("Item"),
    WORLD("World");

    public final String name;

    Category(String name) {
        this.name = name;
    }
}

File: src/main/kotlin/com/skidders/sigma/module/Category.kt
package com.skidders.sigma.module

enum class Category {
    GUI("Gui"),
    MOVEMENT("Movement"),
    RENDER("Render"),
    COMBAT("Combat"),
    MISC("Misc"),
    PLAYER("Player"),
    ITEM("Item"),
    WORLD("World");

    val categoryName: String

    constructor(name: String) {
        this.categoryName = name
    }
}
